src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,7 +66,7 @@ const renderer = ({
       showMore={showMoreConfig !== null}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
-      searchFacetValues={searchForFacetValues && searchForItems}
+      searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

Comment:
maybe there's a way to just _not_ set the prop? 

Can the spread operator help with that or is it overkill